src/integration/pages/articlePage/mundo.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @pathname /mundo/articles/ce42wzqr2mko
+ */
+
+import runCanonicalUserTests from './user.canonical';
+import runAmpUserTests from './user.amp';
+import {
+  runFooterTests,
+  runCommonSeoTests,
+  runCommonCanonicalAnalyticsTests,
+  runCommonA11yTests,
+} from '../../common';
```

Comment:
I like how you explicitly pull the common stuff into each page type rather, than using condtionals like in the cypress test. Open-Closed principle 👍

Reply:
I like how you point out principles and best practices even when I don't intentionally implement them 😄